docs/release_notes/v1.9.5.md

```diff
@@ -0,0 +1,19 @@
+# Dapr 1.9.5
+
+### Fixes an issue in the CosmosDB state store component when performing transaction request with ETags
+
+#### Problem
+
+When setting any ETag in a transaction request inside CosmosDB, dapr exits due to a nil pointer dereference.
```

Comment:
When setting any ETag in a transaction request in the Azure Cosmos DB state store, Dapr panics.

Reply:
Note also the space in "Cosmos DB"

_(that's what you get for having a reviewer who was a former marketing guy :) )_